### PR TITLE
update debian task and add parameter for proxy configuration

### PR DESCRIPTION
**Pull request by atomic111**
Signed-off-by: Patrick Münch <patrick.muench1111@gmail.com>

### CODE REVIEW

File: tasks/pkg_debian.yml [hunk 1]
```diff
@@ -6,17 +6,23 @@
...
 
+- name: Pack Mondoo gpg key
+  command: gpg --dearmor --batch --yes --output {{ mondoo_deb_gpgkey_dest }} /etc/apt/trusted.gpg.d/mondoo.asc
+  become: true
+
 - name: configure mondoo deb repository
   apt_repository:
-    repo: "{{ mondoo_deb_repo }}"
+    repo: "deb [signed-by={{ mondoo_deb_gpgkey_dest }}] {{ mondoo_deb_repo }}"
+    filename: "mondoo"
```
Comment (czunker) [inline tasks/pkg_debian.yml:25]: nit: On my system, I have a `.list` at the end. Should we add this here also?

File: README.md [hunk 2]
```diff
@@ -77,6 +77,24 @@ In addition we support the following variables:
         ensure_managed_client: true
 ```
 
+If you want to use mondoo behind a proxy
+
+```yaml
+---
+- hosts: mondoo_linux
+  become: yes
+  proxy_env:
```
Comment (czunker) [inline README.md:86]: I think, here is a `vars:` missing.

Comment (atomic111) [reply README.md]: thanks for the hint. fixed

File: tasks/pkg_debian.yml (hunk 1 above)
Comment (atomic111) [reply tasks/pkg_debian.yml:26]: ansible adds the `.list` automatic

**Review by czunker — APPROVED** [review]

**Pull request merged**